vec-rdf/vec-rdf-common/pom.xml

```diff
@@ -0,0 +1,47 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.foursoft.harness.vec.rdf</groupId>
+        <artifactId>vec-rdf-parent</artifactId>
+        <version>5.3.1-SNAPSHOT</version>
+    </parent>
+
+    <name>VEC RDF Common</name>
+    <artifactId>vec-rdf-common</artifactId>
+    <description>Common Code for the VEC RDF modules.</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.foursoft.harness.vec</groupId>
```

Comment:
I don't think we should force the user of the RDF module to have all three (and more) standard versions as direct dependencies. We should design this in a way, that the caller adds the needed vec (or kbl) version to it's class path and we do some auto detection or otherwise reflection based approach.